tests/api/conftest.py

```diff
@@ -82,3 +84,8 @@ def staff_api_client(staff_user):
 @pytest.fixture
 def user_api_client(customer_user):
     return ApiClient(user=customer_user)
+
+
+@pytest.fixture
+def anonymous_api_client():
```

Comment:
To keep the convention, it can be just "api_client"

Reply:
Also, please add an issue about changing the user_api_client to api_client in tests